### PR TITLE
Alignment of explainer with the spec 

### PR DESCRIPTION
**Pull request by espinr**
Explainer modified with some **editorial changes** and **alignment with the current discussions** and status of the specification.  

Closes: #48

### CODE REVIEW

**Comment by espinr** [comment]
See the file at https://github.com/espinr/miniapp-packaging/blob/config-file-explainer/docs/explainer.md

File: docs/explainer.md [hunk 1]
```diff
@@ -1,87 +1,91 @@
...
 
-It can be viewed as a normal `.zip` file in some cases (a ZIP parser may open and read the MiniApp package without knowing the MiniApp Packaging spec). However, having the dedicated file extension name and MINE Type gives a way for a hosting platform or a package loader to do pre-checking or content-type negotiation before actually loading/downloading the package.
+Following the common practice of mobile app packaging (such as Android APK), the MiniApp package is designed as a specialized ZIP file, which has its file extension name (`.ma`) and a dedicated MIME type (`application/miniapp-pkg+zip` - to be registered with IANA).
+
+The package can be treated as a regular `.zip` file in some cases (i.e., a ZIP parser may open and read the MiniApp package without following the MiniApp Packaging spec). However, having the dedicated file extension name and MINE Type gives a way for a hosting platform or a package loader to do pre-checking or content-type negotiation before loading/downloading the package.
```
Comment (xfq) [inline docs/explainer.md:36]:
```suggestion
The package can be treated as a regular `.zip` file in some cases (i.e., a ZIP parser may open and read the MiniApp package without following the MiniApp Packaging spec). However, having the dedicated file extension name and MIME Type gives a way for a hosting platform or a package loader to do pre-checking or content negotiation before loading/downloading the package.
```

File: docs/explainer.md [hunk 2]
```diff
@@ -1,87 +1,91 @@
...
+A MiniApp package should guarantee integrity and trustworthiness in the app distribution. So, a MiniApp package may include one or more digital signatures with integrity mechanisms and certificates that enable verifying authors (e.g., the MiniApp developer) and distributors (e.g., an application marketplace). (See more details in the [spec](https://www.w3.org/TR/miniapp-packaging/#sec-miniapp-digital-signature-requirements))
 
-The detailed solution is under development. 
+User agents may require specific digital signature mechanisms, depending on the use case and MiniApp environment, so the specification does not recommends concrete encryption methods or technologies. The specification will include informative examples of various technologies to be used, such as [RFC5652](https://tools.ietf.org/html/rfc5652) (PKCS#7) and the [APK Signature Scheme v2 (https://source.android.com/security/apksigning/v2)).
```
Comment (xfq) [inline docs/explainer.md:57]:
```suggestion
User agents may require specific digital signature mechanisms, depending on the use case and MiniApp environment, so the specification does not recommends concrete encryption methods or technologies. The specification will include informative examples of various technologies to be used, such as [RFC5652](https://tools.ietf.org/html/rfc5652) (PKCS#7) and the [APK Signature Scheme v2](https://source.android.com/security/apksigning/v2)).
```

File: docs/explainer.md [hunk 3]
```diff
@@ -1,87 +1,91 @@
...
+4. **[LPF](https://www.w3.org/TR/lpf/):** It is a lightweight ZIP-based packaging format that contains digital publication materials. It mandates publication-specific files (i.e., `publication.json`, `index.html`) that are not needed by a MiniApp, while misses MiniApp specific files like `manifest.json` and `app.js`. Also, it does not cover the security concerns, such as the integrity protection of the whole package that a MiniApp addresses.
 
-A detailed comparison between MiniApp package and other packaging technologies is given in the following table:
+The following table shows a detailed comparison between a MiniApp package and other packaging technologies
```
Comment (xfq) [inline docs/explainer.md:77]:
```suggestion
The following table shows a detailed comparison between a MiniApp package and other packaging technologies:
```

This is not the caption/title of the table, but a sentence, so punctuation is needed at the end of the sentence.

**Review by xfq — APPROVED** [review]

**Comment by xfq** [comment]
Merging per https://www.w3.org/2022/07/28-miniapp-minutes.html#t03

**Pull request merged**